src/adaptors/http1/http1_server.c

```diff
@@ -712,6 +713,8 @@ static int _server_rx_response_cb(h1_codec_request_state_t *hrs,
            "[C%"PRIu64"][L%"PRIu64"] HTTP response received: status=%d phrase=%s version=%"PRIi32".%"PRIi32,
            hconn->conn_id, hconn->in_link_id, status_code, reason_phrase ? reason_phrase : "<NONE>",
            version_major, version_minor);
+    hreq->base.stop = qd_timer_now();
```

Comment:
Where is this timer started?

Reply:
Rats! I missed the start on the server in the refactoring... it should be in _create_request_context when the hreq is created.